upgrading/automated_upgrades.adoc

```diff
@@ -580,6 +580,15 @@ to ensure you use the latest upgrade playbooks.
 . Ensure the `openshift_deployment_type` parameter in your inventory file is set
 to `openshift-enterprise`.
 
+. Add the `openshift_image_tag` variable to your inventory file. Set its value
+to the full release version, for example `3.11.72`, which you can obtain from
```

Comment:
I think the tag version should have 'v' prefix, such as: 'v3.11.72'

Reply:
'v3.11.x' is correct. 
Thanks.
take real released image tags might be better, such as v3.11.88, v3.10.127

Reply:
I've changed this to v3.11.98 (which released today).